Gopkg.toml

```diff
@@ -12,8 +12,8 @@ required = [
 # Constraints
 
 [[constraint]]
-  branch = "dev"
   name = "github.com/ligato/cn-infra"
+  version = "2.1"
```

Comment:
This would change cn-infra to 2.1 for dev branch in vpp-agent which we don't want, since we would have to switch back to dev after merging to master. You should instead just update cn-infra to latest dev here. And this change of cn-infra version to 2.1 should happen on release branch `2.1.x`, which will be together with changelog merged to master without polluting dev branch.

Reply:
changed